testing/run-scripts/run_cloud.sh

```diff
@@ -153,25 +153,16 @@ HOST_IP=`ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1`
 
 # Start Zork, if necessary.
 if ! docker ps -a | grep uproxy-zork >/dev/null; then
-  if ! docker images | grep uproxy/$1 >/dev/null; then
-    BROWSER=$(echo $1 | cut -d - -f 1)
-    VERSION=$(echo $1 | cut -d - -f 2)
-    IMAGEARGS=
-    if [ -n "$PREBUILT" ]
-    then
-      IMAGEARGS="-p"
-    fi
-    ${BASH_SOURCE%/*}/image_make.sh $IMAGEARGS $BROWSER $VERSION
-  fi
-  HOSTARGS=
-  if [ -n "$PREBUILT" ]
-  then
-    HOSTARGS="$HOSTARGS -v $PREBUILT:/test/src/uproxy-lib"
+  if ! docker images | grep $IMAGE >/dev/null; then
+    ZORK_DIR=`mktemp -d`
+    cp -R ${BASH_SOURCE%/*}/../integration/test $ZORK_DIR/test
+    echo "FROM $IMAGE" > $ZORK_DIR/Dockerfile
+    docker build -t $IMAGE $TMP_DIR
   fi
   # NET_ADMIN is required to run iptables inside the container.
   # Full list of capabilities:
   #   https://docs.docker.com/engine/reference/run/#runtime-privilege-linux-capabilities-and-lxc-configuration
-  docker run --restart=always --net=host --cap-add NET_ADMIN $HOSTARGS --name uproxy-zork -d uproxy/$1 /test/bin/load-zork.sh -z
+  docker run --restart=always --net=host --cap-add NET_ADMIN --name uproxy-zork -d uproxy/$1 /test/bin/load-zork.sh -z
```

Comment:
I just realized the `uproxy/$1` depended on the browser-version argument, and since I'm dropping that I should put something else here. Not sure what to put though, could just be $IMAGE, thoughts welcome.